wrappers/python/simtk/openmm/app/gromacstopfile.py

```diff
@@ -591,7 +596,14 @@ def createSystem(self, nonbondedMethod=ff.NoCutoff, nonbondedCutoff=1.0*unit.nan
             sys.setDefaultPeriodicBoxVectors(*boxVectors)
         elif nonbondedMethod in (ff.CutoffPeriodic, ff.Ewald, ff.PME, ff.LJPME):
             raise ValueError('Illegal nonbonded method for a non-periodic system')
-        nb = mm.NonbondedForce()
+        if self._defaults[1] == '1':
+            # factor of 138.935456 for coulomb taken from tests/TestCustomNonbondedForce.h:574
+            nb = mm.CustomNonbondedForce('138.935456*q1*q2/r-sqrt(C1*C2)/r^6+sqrt(A1*A2)/r^12')
```

Comment:
Hi @swalis.  To be honest, in our models we don't need to include electrostatics (q = 0 everywhere).  I thought this would be nice to attempt to support electrostatics for future users.  I suppose I could implement the split method you suggested.